client/my-sites/plugins/plugin-ratings/index.jsx

```diff
@@ -105,9 +113,18 @@ class PluginRatings extends Component {
 			<div className="plugin-ratings">
 				<div className="plugin-ratings__rating-stars">
 					<Rating rating={ rating } />
-					<span className="plugin-ratings__number">{ rating / 20 }</span>
+					{ inlineNumRatings && numRatings && (
+						<span className="plugin-ratings__num-ratings">
+							(
+							{ rating && Number.isInteger( numRatings )
```

Comment:
Noting that the `rating &&` check here is unnecessary because we already have a check that ensures `rating` is not falsy on L107.

Reply:
Thanks for pointing it. 

I created a follow-up PR with this change #59119 59119

Reply:
It wasn't urgent, but thanks for the super swift turnaround 🚀